src/SablierMerkleVCA.sol

```diff
@@ -138,24 +145,44 @@ contract SablierMerkleVCA is
     //////////////////////////////////////////////////////////////////////////*/
 
     /// @dev See the documentation for the user-facing functions that call this internal function.
-    function _calculateClaimAmount(uint128 fullAmount, uint40 claimTime) internal view returns (uint128) {
-        // Calculate the claim amount.
-        uint40 elapsedTime;
-        uint40 totalDuration;
+    function _calculateClaimAmount(uint128 fullAmount, uint40 claimTime) internal view returns (uint128 claimAmount) {
+        // If the claim time is less than start time, return zero.
+        if (claimTime < START_TIME) {
+            return 0;
+        }
+
+        // Calculate the initial unlocked amount.
+        uint128 initialUnlockedAmount = ud(fullAmount).mul(UNLOCK_PERCENTAGE).intoUint128();
+
+        // If claim time is equal to the start time, return the initial unlock.
+        if (claimTime == START_TIME) {
+            return initialUnlockedAmount;
+        }
 
         // If the vesting period has ended, the claim amount is the full amount.
         if (claimTime >= END_TIME) {
             return fullAmount;
         }
         // Otherwise, calculate the claim amount based on the elapsed time.
         else {
+            uint40 elapsedDuration;
+            uint40 totalDuration;
+            uint128 vestingFullAmount;
+
+            // Safe to unchecked because it cannot overflow due to above checks.
             unchecked {
-                elapsedTime = claimTime - START_TIME;
+                elapsedDuration = claimTime - START_TIME;
                 totalDuration = END_TIME - START_TIME;
+                vestingFullAmount = fullAmount - initialUnlockedAmount;
```

Comment:
There might be bugs in PRBMath which lead to the `initialUnlockedAmount` being higher than `fullAmount`.